gpep517/__main__.py

```diff
@@ -322,6 +327,14 @@ def add_build_args(parser):
                        help="Patch sysconfig paths to use specified sysroot "
                             "(experimental cross-compilation support)",
                        type=Path)
+    group.add_argument("--broot",
+                       help="Prefix applied to the build host (only "
+                            "applicable with --sysroot option)",
+                       type=str)
```

Comment:
`type=str` is redundant.

Reply:
I was making it clear it's not a `Path` but okay. I would have used `Path`, but giving an empty string results in `Path(".")`, which is annoying. We avoid that problem in the eclass by not giving `--sysroot` when it would be blank, but I feel like it should just do the right thing here instead, either by using `str` instead or by checking for `Path(".")`. What do you think?

Reply:
I suppose you could just `default=Path("/")` but I don't have a strong opinion. This could deal better with double slashes and stuff.

Reply:
`default` is only used when the option is not given at all, not when it's a blank string. I tried your suggestion and it still ends up with `Path(".")`. What we have is fine as long as the eclass uses it correctly, so let's just go with that.